<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1026</width>
    <height>778</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>(T)Raum - Plan dein Jugendzimmer 2.0</string>
  </property>
  <property name="windowIcon">
   <iconset resource="Images/selecttool.qrc">
    <normaloff>:/new/prefix1/ic_add_box_black_48dp.png</normaloff>:/new/prefix1/ic_add_box_black_48dp.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">QMainWindow {
     background-color: #FFFDB5;
}</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="styleSheet">
    <string notr="true">QWidget {
     background-color: #FFFDB5;
}</string>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="1" column="3" rowspan="3">
       <layout class="QGridLayout" name="gridLayout_5">
        <item row="1" column="0" rowspan="2">
         <widget class="ObjectSettings" name="objectSettingsWidget" native="true">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>27</height>
           </size>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QGraphicsView" name="graphicsView">
          <property name="cursor" stdset="0">
           <cursorShape>ArrowCursor</cursorShape>
          </property>
          <property name="styleSheet">
           <string notr="true">QWidget {
	background-color: #FFFFFF; border: 1px solid black;
	border-radius: 5px;
}</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="3" column="2">
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="5" column="0">
         <layout class="QGridLayout" name="gridLayout_4">
          <item row="1" column="0">
           <widget class="QGroupBox" name="settingsBox">
            <property name="title">
             <string>Größe und Position</string>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout_2">
             <item>
              <layout class="QFormLayout" name="formLayout">
               <item row="0" column="1">
                <widget class="QLabel" name="lb_x_pos">
                 <property name="toolTip">
                  <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt;&quot;&gt;Hier kannst Du einstellen, wie weit rechts, oder Links das Möbelstück hingestellt werden soll. Das kannst du aber später noch ändern.&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
                 </property>
                 <property name="text">
                  <string>Position (R/L) </string>
                 </property>
                </widget>
               </item>
               <item row="1" column="1">
                <widget class="QLabel" name="lb_y_pos">
                 <property name="toolTip">
                  <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt;&quot;&gt;Hier kannst Du einstellen, wie weit vorne oder hinten das Möbelstück hingestellt werden soll. Auf deinem Bildschirm wird es als Oben und Unten dargestellt. Das kannst du aber später noch ändern.&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
                 </property>
                 <property name="text">
                  <string>Position (V/H)</string>
                 </property>
                </widget>
               </item>
               <item row="2" column="0">
                <widget class="QSpinBox" name="sb_width">
                 <property name="maximum">
                  <number>499</number>
                 </property>
                 <property name="value">
                  <number>10</number>
                 </property>
                </widget>
               </item>
               <item row="3" column="1">
                <widget class="QLabel" name="lb_width">
                 <property name="text">
                  <string>cm Breite</string>
                 </property>
                </widget>
               </item>
               <item row="2" column="1">
                <widget class="QLabel" name="lb_height">
                 <property name="text">
                  <string>cm Länge</string>
                 </property>
                </widget>
               </item>
               <item row="3" column="0">
                <widget class="QSpinBox" name="sb_heigth">
                 <property name="maximum">
                  <number>499</number>
                 </property>
                 <property name="value">
                  <number>10</number>
                 </property>
                </widget>
               </item>
               <item row="0" column="0">
                <widget class="QDoubleSpinBox" name="sb_x_pos">
                 <property name="toolTip">
                  <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt;&quot;&gt;Hier kannst Du einstellen, wie weit rechts, oder Links das Möbelstück hingestellt werden soll. Das kannst du aber später noch ändern.&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
                 </property>
                 <property name="decimals">
                  <number>0</number>
                 </property>
                 <property name="maximum">
                  <double>1600.000000000000000</double>
                 </property>
                </widget>
               </item>
               <item row="1" column="0">
                <widget class="QDoubleSpinBox" name="sb_y_pos">
                 <property name="toolTip">
                  <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt;&quot;&gt;Hier kannst Du einstellen, wie weit vorne oder hinten das Möbelstück hingestellt werden soll. Auf deinem Bildschirm wird es als Oben und Unten dargestellt. Das kannst du aber später noch ändern.&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
                 </property>
                 <property name="decimals">
                  <number>0</number>
                 </property>
                 <property name="maximum">
                  <double>1000.000000000000000</double>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QPushButton" name="pb_AddObject">
            <property name="autoFillBackground">
             <bool>false</bool>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton {
     background-color: #06E100; border: 1px solid black;
     border-radius: 5px;
}</string>
            </property>
            <property name="text">
             <string>Ab ins Zimmer damit</string>
            </property>
            <property name="icon">
             <iconset resource="Images/selecttool.qrc">
              <normaloff>:/new/prefix1/ic_done_black_24dp.png</normaloff>:/new/prefix1/ic_done_black_24dp.png</iconset>
            </property>
            <property name="checkable">
             <bool>false</bool>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item row="1" column="2">
       <widget class="DrawingToolSelector" name="drawingToolSelector" native="true"/>
      </item>
      <item row="2" column="2">
       <widget class="ColorToolSelector" name="colorToolSelector" native="true"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1026</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="actionSave"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuTool">
    <property name="title">
     <string>&amp;Tool</string>
    </property>
    <addaction name="actionSelect_Tool"/>
    <addaction name="separator"/>
    <addaction name="actionLine_Tool"/>
    <addaction name="actionRectangle_Tool"/>
    <addaction name="actionCircle_Tool"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuTool"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionOpen">
   <property name="text">
    <string>&amp;Open</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>&amp;Save</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionSelect_Tool">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="Images/selecttool.qrc">
     <normaloff>:/new/prefix1/ic_pan_tool_black_48dp.png</normaloff>:/new/prefix1/ic_pan_tool_black_48dp.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Select Tool</string>
   </property>
   <property name="shortcut">
    <string>Alt+S</string>
   </property>
  </action>
  <action name="actionCircle_Tool">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset>
     <normalon>:/new/prefix1/ic_fiber_manual_record_black_48dp.png</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>&amp;Circle Tool</string>
   </property>
   <property name="shortcut">
    <string>Alt+C</string>
   </property>
  </action>
  <action name="actionRectangle_Tool">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset>
     <normalon>:/new/prefix1/ic_stop_black_48dp.png</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>&amp;Rectangle Tool</string>
   </property>
   <property name="shortcut">
    <string>Alt+R</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>&amp;Exit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionLine_Tool">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="Images/selecttool.qrc">
     <normaloff>:/new/prefix1/ic_timeline_black_48dp.png</normaloff>:/new/prefix1/ic_timeline_black_48dp.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Line Tool</string>
   </property>
   <property name="shortcut">
    <string>Alt+L</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>DrawingToolSelector</class>
   <extends>QWidget</extends>
   <header>drawingtoolselector.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ColorToolSelector</class>
   <extends>QWidget</extends>
   <header>colortoolselector.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ObjectSettings</class>
   <extends>QWidget</extends>
   <header>objectsettings.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="Images/selecttool.qrc"/>
 </resources>
 <connections/>
</ui>
